### PR TITLE
WIP docs for the new Explore feature

### PR DESCRIPTION
**Pull request by daniellee**
This is a very rough draft of the docs for Explore. Need to add links and images but will wait with images and gifs until Explore development has gotten further.

### CODE REVIEW

File: docs/sources/features/explore/index.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+weight = 5
++++
+
+# Introduction
+
+One of the major new features of Grafana 6.0 is the new query-focused Explore workflow for troubleshooting and/or for data exploration.
+
+Grafana's dashboard UI is all about building dashboards for visualization. Explore strips away all the dashboard and panel options so that you can focus on the query. Iterate until you have a working query and then think about building a dashboard.
+
+For infrastructure monitoring, responding to an incident is when you usually leave Grafana and use other tools to debug what went wrong. Explore allows you to dig deeper into your metrics and logs to find the cause. The new logging feature in Explore correlates metrics and logs and allows for a tight integration in Grafana. This creates a new debugging workflow where you can:
```
Comment (davkal) [inline docs/sources/features/explore/index.md]: More positive:
For infrastructure monitoring and incident response, you no longer need to switch to other tools to debug what went wrong.

**Comment by daniellee** [comment]
Added some more text, but still have some empty areas. For example, how text searching works for logs.

File: docs/sources/features/explore/index.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+weight = 5
++++
+
+# Introduction
+
+One of the major new features of Grafana 6.0 is the new query-focused Explore workflow for troubleshooting and/or for data exploration.
+
+Grafana's dashboard UI is all about building dashboards for visualization. Explore strips away all the dashboard and panel options so that you can focus on the query. Iterate until you have a working query and then think about building a dashboard.
+
+For infrastructure monitoring and incident response, you no longer need to switch to other tools to debug what went wrong. Explore allows you to dig deeper into your metrics and logs to find the cause. The new logging feature in Explore correlates metrics and logs and allows for a tight integration in Grafana. This creates a new debugging workflow where you can:
```
Comment (davkal) [inline docs/sources/features/explore/index.md]: Misleading: the product does not correlate, you can.
"Grafana's new logging datasource is tightly integrated into Explore and allows you to correlate metrics and logs side-by-side."

File: docs/sources/features/explore/index.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+1. Receive an alert
+2. Drill down and examine metrics
+3. Drill down again and search logs related to the metric and time interval (and in the future, distributed traces).
+
+But if you just want to explore your data and do not want to create a dashboard then Explore makes this much easier. Explore will also if possible show both graph and table results.
```
Comment (davkal) [inline docs/sources/features/explore/index.md]: s/But/Even/

File: docs/sources/features/explore/index.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+## Split and Compare
+
+The Split feature is an easy way to compare graphs and tables side-by-side or to look at related data together on one page. Click the split button to duplicate the current query and split the page into two side-by-side queries. It is possible to select another datasource for the new query which for example, allows you to compare the same query for two different servers or to compare the staging environment to the production environment.
+
+You can close the newly created query by clicking on the Close Split button.
+
+## Prometheus-specific Features
+
+The first version of Explore features a custom querying experience for Prometheus. When a query is executed, it actually executes two queries. One for the graph and an Instant Query for the table. An Instant Query returns the last value for each time series e.g. if it the query results returns
```
Comment (davkal) [inline docs/sources/features/explore/index.md]: Misses `.` at the end.

File: docs/sources/features/explore/index.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+You can close the newly created query by clicking on the Close Split button.
+
+## Prometheus-specific Features
+
+The first version of Explore features a custom querying experience for Prometheus. When a query is executed, it actually executes two queries. One for the graph and an Instant Query for the table. An Instant Query returns the last value for each time series e.g. if it the query results returns 
+
+### Metrics Explorer
+
+On the left-hand side of the query field is a `Metrics` button, clicking on this opens the Metric Explorer. This shows a hierarchical menu with metrics grouped by their prefix. For example, all the Alert Manager metrics will be grouped under the `alertmanager` prefix. This is a good starting point if you just want to explore which metrics are available.
+
```
Comment (davkal) [inline docs/sources/features/explore/index.md]: "Prometheus only: The Metrics Explorer also has a short-cut to all your recording rules at the top, as well as a list of all metrics that support histograms."

File: docs/sources/features/explore/index.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+- For buckets, a histogram function will be suggested.
+- For recording rules, possible to expand the rules.
+
+### Table Filters
+
+Click on the cell links in the Table panel to add filters to the query expression. This works with multiple queries too - the filter will be added for all the queries.
+
+### Logs Integration
+
+For Grafana 6.0, the first log integration is for the new open source log aggregation system from Grafana Labs - Loki. Loki is designed to be very cost effective, as it does not index the contents of the logs, but rather a set of labels for each log stream. The logs from Loki are queried in a similar to metric queries in Prometheus. It uses labels to group log streams which can be made to match up with your Prometheus labels. Read more about Loki here or the Grafana Labs hosted variant: Grafana Cloud Logs.
```
Comment (davkal) [inline docs/sources/features/explore/index.md]: ...queried in a similar way as you would Prometheus for metrics.

File: docs/sources/features/explore/index.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+If you switch from a Prometheus query to a logs query (you can do a split first to have your metrics and logs side by side) then it will keep the labels from your query and use those to query the log streams. For example, the following Prometheus query:
```
Comment (davkal) [inline docs/sources/features/explore/index.md]: ...will keep the labels from your query that exist in the logs, and use those...

File: docs/sources/features/explore/index.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+This will return a chunk of logs in the selected time range that can be grepped/text searched.
+
+## Log Features
+
+Explore for logs currently integrates with Loki and Grafana Cloud Logs. Create a new Grafana Logging datasource in Grafana ...
+
+### Log Queries
+
+The Metrics Explorer (the `Log labels` button) beside the query field shows a list of labels that you can use to filter your logs. An alternative way to write a query is to use the query field's autocomplete - you start by typing a left curly brace `{` and the autocomplete menu will suggest a list of labels. Press the `enter` key to execute the query.
```
Comment (davkal) [inline docs/sources/features/explore/index.md]: The Logs Explorer

**Review by marefr — COMMENTED** [review]
Maybe a note about current "beta" state and how you can enable explore using feature flag?

File: docs/sources/features/explore/index.md (hunk 4 above)
Comment (daniellee) [reply docs/sources/features/explore/index.md]: Yeah, unfinished sentence. Looking for good docs on instant queries.

File: docs/sources/features/explore/index.md (hunk 5 above)
Comment (daniellee) [reply docs/sources/features/explore/index.md]: Do I need to add that? It is a subsection in the Prometheus-specific section.

File: docs/sources/features/explore/index.md (hunk 3 above)
Comment (daniellee) [reply docs/sources/features/explore/index.md]: Hmm, don't like that how this sentence reads. Will rewrite the whole sentence.

File: docs/sources/features/explore/index.md (hunk 7 above)
Comment (daniellee) [reply docs/sources/features/explore/index.md]: Ha, didn't know that it was that smart. Thanks.

File: docs/sources/features/explore/index.md (hunk 8 above)
Comment (daniellee) [reply docs/sources/features/explore/index.md]: Thanks for this - was going to ask about the query language syntax.

File: docs/sources/features/explore/index.md (hunk 6 above)
Comment (daniellee) [reply docs/sources/features/explore/index.md]: Changed it to: *The logs from Loki are queried in a similar way to querying with label selectors in Prometheus*

File: docs/sources/features/explore/index.md (hunk 3 above)
Comment (daniellee) [reply docs/sources/features/explore/index.md]: Left out the But and rewrote the next sentence.

**Comment by daniellee** [comment]
@davkal 

Preview here: http://staging-docs.grafana.org/features/explore/

todo:

- [ ] Add instructions about feature flag
- [ ] Make images look nicer

**Pull request merged**